packages/react-composites/src/composites/common/icons.tsx

```diff
@@ -83,7 +83,7 @@ export const DEFAULT_COMPOSITE_ICONS = {
 };
 
 /** @private */
-export type CompositeIconProps<Icons> = IIconProps & { iconName: keyof Icons };
+export type CompositeIconProps<Icons> = IIconProps | { iconName: keyof Icons };
```

Comment:
Changing this to a OR union is to make it so contoso can provide any string here correct?